Main.js

```diff
@@ -1,12 +1,30 @@
+const fs = require('fs');
 const { readFile } = require('./readFile.js');
 const { uploadFile } = require('./uploadFile.js');
+const { isFile } = require('./isFile.js');
 
-const gistName = process.argv[3];
-const file = process.argv[2];
+const contentAndNameList = process.argv;
```

Comment:
always define variables in the narrowest scope as possible.

There 3 variables should be defined inside `Runner` function

Reply:
done